bin/dockerfilelint

```diff
@@ -4,14 +4,36 @@ var process = require('process');
 var fs = require('fs');
 var os = require("os");
 var path = require('path');
-var argv = require('yargs').argv;
+var usage = 'Usage: dockerfilelint [files | content..] [options]';
+var argv = require('yargs')
+             .usage(usage)
```

Comment:
No worries! I usually like to see things near each other that are chained and it was just a habit of mine :)